verification/getProfile_test.go

```diff
@@ -10,40 +10,48 @@ import (
 // This file is used to test the get profile command by using a simulator
 
 func TestGetProfile(t *testing.T) {
-	simulators := []TestDPEInstance{
-		// No extra options.
-		&DpeSimulator{exe_path: *sim_exe},
-		// Supports simulation.
-		&DpeSimulator{exe_path: *sim_exe, supports: Support{Simulation: true}},
-		// Supports extended TCI.
-		&DpeSimulator{exe_path: *sim_exe, supports: Support{ExtendTci: true}},
-		// Supports auto-init.
-		&DpeSimulator{exe_path: *sim_exe, supports: Support{AutoInit: true}},
-		// Supports tagging.
-		&DpeSimulator{exe_path: *sim_exe, supports: Support{Tagging: true}},
-		// Supports rotate context.
-		&DpeSimulator{exe_path: *sim_exe, supports: Support{RotateContext: true}},
-		// Supports certify key.
-		&DpeSimulator{exe_path: *sim_exe, supports: Support{X509: true}},
-		// Supports certify csr.
-		&DpeSimulator{exe_path: *sim_exe, supports: Support{Csr: true}},
-		// Supports symmetric derivation.
-		&DpeSimulator{exe_path: *sim_exe, supports: Support{IsSymmetric: true}},
-		// Supports internal info.
-		&DpeSimulator{exe_path: *sim_exe, supports: Support{InternalInfo: true}},
-		// Supports internal DICE.
-		&DpeSimulator{exe_path: *sim_exe, supports: Support{InternalDice: true}},
-		// Supports IsCA
-		&DpeSimulator{exe_path: *sim_exe, supports: Support{IsCA: true}},
-		// Supports a couple combos.
-		&DpeSimulator{exe_path: *sim_exe, supports: Support{Simulation: true, AutoInit: true, RotateContext: true, Csr: true, InternalDice: true, IsCA: true}},
-		&DpeSimulator{exe_path: *sim_exe, supports: Support{ExtendTci: true, Tagging: true, X509: true, InternalInfo: true}},
-		// Supports everything.
-		&DpeSimulator{exe_path: *sim_exe, supports: Support{Simulation: true, ExtendTci: true, AutoInit: true, Tagging: true, RotateContext: true, X509: true, Csr: true, IsSymmetric: true, InternalInfo: true, InternalDice: true, IsCA: true}},
+	var instances []TestDPEInstance
+	if *isEmulator {
+		//Added dummy support for emulator. Once the emulator is implemented, will add the actual enabled feature
+		instances = []TestDPEInstance{
+			&DpeInstance{exe_path: *socket_exe, supports: Support{AutoInit: true}},
+		}
+	} else {
+		instances = []TestDPEInstance{
```

Comment:
Since testing the different support vectors only works with the simulator, I would recommend splitting this into its own test (something like `TestProfileMatrix`) and skipping it if the test target isn't the simulator.